packages/wrangler/src/docs/index.ts

```diff
@@ -29,12 +30,12 @@ export async function docsHandler(
 	//if no command is provided, open the docs homepage
 	//or, if a command IS provided, but we can't find anything, open the docs homepage
 	let urlToOpen =
-		args.yes || !args.command || args.command.length === 0
+		args.yes || !args.search || args.search.length === 0
 			? "https://developers.cloudflare.com/workers/wrangler/commands/"
 			: "";
 
-	if (args.command && args.command.length > 0) {
-		const searchTerm = args.command.join(" ");
+	if (args.search && args.search.length > 0) {
+		const searchTerm = args.search.join(" ");
 		const searchResult = await runSearch(searchTerm);
 
 		urlToOpen = searchResult ?? urlToOpen;
```

Comment:
I find the following less convoluted:

```
	// if no query is provided, open the docs homepage
	// or, if a query IS provided, but we can't find anything, open the docs homepage
  let urlToOpen = "https://developers.cloudflare.com/workers/wrangler/commands/";

	if (args.search?.length > 0) {
		const searchTerm = args.search.join(" ");
		const searchResult = await runSearch(searchTerm);

    if (searchResult) {
      urlToOpen = searchResult;
    } else if (!arg.yes) {
      urlToOpen = undefined;   
    }
	} 
```

One nit (because it's not introduced in this PR) is that `runSearch` could be a more descriptive name, i.e. where are we searching (wrangler, cloudflare, .... docs) and what do we return ("url"). It also doesn't have comment

Reply:
Yeah I generally agree that this code could be a bit cleaner but I didn't think it was too bad and didn't want to churn the code too much here.